.travis/custom_check_pull_request.py

```diff
@@ -56,23 +56,23 @@ def git_commit_message(commit_sha):
 def check_changelog_record(issue):
     reconrds_count = len(glob.glob(f"CHANGES/{issue}.*"))
     if reconrds_count == 0:
-        LOG.error(f"Missing change log entry for issue #{issue}.")
+        LOG.error(f"Missing change log entry for issue AAH-{issue}.")
         return False
     if reconrds_count == 1:
         return True
-    LOG.error(f"Multiple change log records found for issue #{issue}.")
-    return False
+    LOG.info(f"Multiple change log records found for issue AAH-{issue}.")
```

Comment:
Can you please explain why the logic of this check got updated?

Reply:
@cutwater as we are starting the counting again on Jira there will be duplicate issue numbers, for example: There is a #44 here on Github and also an AAH-44 on Jira.

Reply:
@rochacbruno these files get deleted on each generation of the updated `CHANGELOG`, so there shouldn't be any old files related to GitHub, and since we are moving forward with Jira there shouldn't be any new files related to GitHub, so we can probably keep this as `error` and `return False`